webapp/src/sui.tsx

```diff
@@ -82,13 +90,29 @@ export class DropdownMenuItem extends UiElement<DropdownProps> {
     componentDidMount() {
         this.popup()
         this.child("").dropdown({
-            action: "hide",
+            action: (text: string, value: any, element: any) => {
```

Comment:
The reason why is that depending of if we click on a href or a div, the behavior of .click() is different, and we call "action" two times in some cases. One time with a Jquery element, and the other time with the contained HTMLElement of this Jquery value.